doc/reference/configuration/cfg_logging.rst

```diff
@@ -32,7 +36,7 @@
     | Dynamic: **yes**
 
     Warning: prior to Tarantool 1.7.5 there were only six levels and ``DEBUG`` was
```

Comment:
Maybe turn this into a real [warning](https://www.tarantool.io/en/doc/latest/contributing/docs/markup/admonitions/)?

Reply:
Converted to a note